src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -20,16 +20,18 @@
             [status-im2.common.not-implemented :as not-implemented]
             [utils.datetime :as datetime]
             [reagent.core :as reagent]
-            [utils.address :as address]))
+            [utils.address :as address]
+            [quo.gesture-handler :as gesture-handler]))
 
 (def delivery-state-showing-time-ms 3000)
 
 (defn avatar-container
-  [{:keys [content last-in-group? pinned-by quoted-message from]}]
+  [{:keys [content last-in-group? pinned-by quoted-message from]} is-message-reaction-view?]
```

Comment:
`is-` is unnecessary with the question mark.

Reply:
Removed